.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -1,5 +1,5 @@
-name: 💡 Feature request
-description: Suggest an idea for this project
+name: 💡 Feature Request
+description: Suggest an idea for this project or submit a new page
 title: "[💡]: "
 labels: ["support", "triage"]
```

Comment:
Previously, it was mentioned that each PR would have an Issue associated with it (to link in the PR). Is the feature request the same as a issue template for new pages? I assumed that the Feature Request is the same as asking for a new page, but are they separate?

Reply:
Ahh, I can see your reasoning now, okay. I have a couple thoughts.

1. Maybe "Feature Request" isn't the best name for this template on this repo. We probably shouldn't have that discussion here, though.
2. I think using issues (and this feature request issue specifically) to "submit" new pages is confusing.
4. We should separate out the goal of having an issue for every PR from always requiring external contributors to create that issue. Also, I imagine that, if we receive a PR from the wild without an issue, it will be easier to make and link an issue ourselves (like, "Investigate incorporating #PR) instead of asking that the contributor make one themselves.

My opinion would be to not edit the issue template in this PR and generally not assume that we're going to get external folks to follow our internal workflows.